src/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+import Page from './Page';
+
+function App() {
+  const [newTodo, setNewTodo] = useState('');
+  const [todos, setTodos] = useState([]);
+
+  const handleChangeTodo = (e) => {
+    setNewTodo(e.target.value);
+  };
+
+  const handleAddTodo = () => {
+    setNewTodo('');
+    setTodos([...todos, newTodo]);
+  };
+
+  const handleRemoveTodo = (deletedTodoIdx) => {
+    setTodos(todos.filter((todo, todoIdx) => todoIdx !== deletedTodoIdx));
```

Comment:
사용되지 않는 것을 이렇게 표현해도 괜찮을 것 같아요
```suggestion
    setTodos(todos.filter((_, todoIdx) => todoIdx !== deletedTodoIdx));
```